data/packages/com.cammin.ldtkunity.yml

```diff
@@ -8,13 +8,14 @@ parentRepoUrl: null
 licenseSpdxId: MIT
 licenseName: MIT License
 topics:
+  - 2d
   - integration
   - level-design
 hunter: JesseTG
 gitTagPrefix: ''
 gitTagIgnore: ''
-minVersion: ''
+minVersion: '2019.3'
```

Comment:
The `minVersion` is not Unity editor min version, but the Git tag min version to build. In your case, please just leave it no change.

Reply:
More about the YAML file: https://openupm.com/docs/adding-upm-package.html#package-yaml-file

Reply:
Thanks for the info 👍